app/src/main/res/values/dimens.xml

```diff
@@ -43,6 +43,7 @@
     <dimen name="dp_25">25dp</dimen>
     <dimen name="dp_180">180dp</dimen>
     <dimen name="sp_25">25sp</dimen>
+    <dimen name="dp_6">6dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_6` appears to be unused
 Unused resources make applications larger and slow down builds.